docs/contributing-to-keyshade/environment-variables.md

```diff
@@ -26,3 +26,4 @@ Here's the description of the environment variables used in the project. You can
 - **JWT_SECRET**: The secret used to sign the JWT tokens. It is insignificant in the development environment.
 - **WEB_FRONTEND_URL, WORKSPACE_FRONTEND_URL**: The URLs of the web and workspace frontend respectively. These are used in the emails sometimes and in other spaces of the application too.
 - **API_PORT**: The environmental variable that specifies the port number on which the API server should listen for incoming connections. If not explicitly set, it defaults to port 4200.
+- **FEEDBACK_FORWARD_EMAIL**: The environment variable that specifies the feedback forward email address
```

Comment:
Set this to - 

> Feedbacks submitted by the user would be sent to this email address for the concerned authorities to view it. Ideally, in development environment, this would be your personal email address.